declarations.d.ts

```diff
@@ -0,0 +1,27 @@
+/*****
```

Comment:
no need for a separated declarations file, if you want to use central-services-shared, copy-paste declarations from thirdparty-[api/scheme]-adapter if you plan to use something which is missed there you need to declare it properly. Empty declarations only postpone in time the moment when declarations have to be written. Empty declarations allow to import module in TS but you can't use imported elements without TS complaints or complex casting.